### PR TITLE
:tv: Add youtube links to talks

### PR DESCRIPTION
**Pull request by jefftriplett**

### CODE REVIEW

File: _schedule/talks/2019-09-23-11-30-t0-awesome-automated-apis-with-automagic.md [hunk 1]
```diff
@@ -35,4 +35,4 @@ Databases stay relevant by continuing to reinvent themselves to serve new techno
...
 
-The models for the ORM, serializers for the RESTful API, views for presenting the data to a user, filters for refining queries, URL routing, web browsable interface, user token authorization, and permissions, are all handled by introspecting various features of the PostgreSQL database information schema, all with Python. Many of these components have been open-sourced, such as the ability to export an endpoint as a spreadsheet file (XLSX), and a generic automated API builder, called Automagic REST.
\ No newline at end of file
+The models for the ORM, serializers for the RESTful API, views for presenting the data to a user, filters for refining queries, URL routing, web browsable interface, user token authorization, and permissions, are all handled by introspecting various features of the PostgreSQL database information schema, all with Python. Many of these components have been open-sourced, such as the ability to export an endpoint as a spreadsheet file (XLSX), and a generic automated API builder, called Automagic REST.
```
Comment (drewbrew) [inline _schedule/talks/2019-09-23-11-30-t0-awesome-automated-apis-with-automagic.md]: what changed here?

File: _schedule/talks/2019-09-23-12-50-t0-lightning-talks.md [hunk 2]
```diff
@@ -12,6 +12,13 @@ room: Salon A-E
 sitemap: true
 title: Lightning Talks
 track: t0
+video_url: https://www.youtube.com/watch?v=b5jS4mLR1N0
 ---
 
-Salon A-E
\ No newline at end of file
+- Wayne Merry - 00:00
```
Comment (drewbrew) [inline _schedule/talks/2019-09-23-12-50-t0-lightning-talks.md]:
```suggestion
- Wayne Merry - [00:00](https://www.youtube.com/watch?v=b5jS4mLR1N0&t=0)
```

If the markdown rendering supports links, let's link directly to the times

File: _schedule/talks/2019-09-23-12-50-t0-lightning-talks.md [hunk 3]
```diff
@@ -12,6 +12,13 @@ room: Salon A-E
...
+- Ayaz Amlani - 5:24
```
Comment (drewbrew) [inline _schedule/talks/2019-09-23-12-50-t0-lightning-talks.md]:
```suggestion
- Ayaz Amlani - [5:24](https://www.youtube.com/watch?v=b5jS4mLR1N0&t=324)
```

File: _schedule/talks/2019-09-23-12-50-t0-lightning-talks.md [hunk 4]
```diff
@@ -12,6 +12,13 @@ room: Salon A-E
...
+- Tyler Rudie - 9:37
```
Comment (drewbrew) [inline _schedule/talks/2019-09-23-12-50-t0-lightning-talks.md]:
```suggestion
- Tyler Rudie - [9:37](https://www.youtube.com/watch?v=b5jS4mLR1N0&t=577])
```

File: _schedule/talks/2019-09-23-12-50-t0-lightning-talks.md [hunk 5]
```diff
@@ -12,6 +12,13 @@ room: Salon A-E
...
+- Pandy Knight - 14:30
```
Comment (drewbrew) [inline _schedule/talks/2019-09-23-12-50-t0-lightning-talks.md]:
```suggestion
- Pandy Knight - [14:30](https://www.youtube.com/watch?v=b5jS4mLR1N0&t=870)
```

File: _schedule/talks/2019-09-23-12-50-t0-lightning-talks.md [hunk 6]
```diff
@@ -12,6 +12,13 @@ room: Salon A-E
...
+- Dustin Ingram - 18:32
```
Comment (drewbrew) [inline _schedule/talks/2019-09-23-12-50-t0-lightning-talks.md]:
```suggestion
- Dustin Ingram - [18:32](https://www.youtube.com/watch?v=b5jS4mLR1N0&t=1112
```

File: _schedule/talks/2019-09-23-12-50-t0-lightning-talks.md [hunk 7]
```diff
@@ -12,6 +12,13 @@ room: Salon A-E
...
+- Paul Stieber - 22:23
```
Comment (drewbrew) [inline _schedule/talks/2019-09-23-12-50-t0-lightning-talks.md]:
```suggestion
- Paul Stieber - [22:23](https://www.youtube.com/watch?v=b5jS4mLR1N0&t=1343)
```

File: _schedule/talks/2019-09-23-12-50-t0-lightning-talks.md (hunk 5 above)
Comment (drewbrew) [reply _schedule/talks/2019-09-23-12-50-t0-lightning-talks.md]: Q: Do we want to address him as Andrew, Andy, or Pandy? (In his talk, he says "I'm Andy")

image

File: _schedule/talks/2019-09-23-12-50-t0-lightning-talks.md [hunk 8]
```diff
@@ -12,6 +12,13 @@ room: Salon A-E
...
+- Dawn Wages - 26:16
```
Comment (drewbrew) [inline _schedule/talks/2019-09-23-12-50-t0-lightning-talks.md]:
```suggestion
- Dawn Wages - [26:16](https://www.youtube.com/watch?v=b5jS4mLR1N0&t=1576)
```

File: _schedule/talks/2019-09-24-12-50-t0-lightning-talks.md [hunk 9]
```diff
@@ -12,8 +12,15 @@ room: Salon A-E
 sitemap: true
 title: Lightning Talks
 track: t0
+video_url: https://www.youtube.com/watch?v=5j9JIUuA_cM
 ---
 
-Salon A-E
-
-Tuesday sign ups go live Monday at 2:20pm and close Tuesday at 9am (please eat breakfast!) with speaker notifications by 11am Tuesday. [Register here!](http://bit.ly/dcus2019lt) 
+- Adem Gaygusuz - 00:00
```
Comment (drewbrew) [inline _schedule/talks/2019-09-24-12-50-t0-lightning-talks.md]:
```suggestion:-0+8
- Adem Gaygusuz - [00:00](https://www.youtube.com/watch?v=5j9JIUuA_cM&t=0)
- Laura Lorenz - [5:24](https://www.youtube.com/watch?v=5j9JIUuA_cM&t=324)
- Micah Lyle - [10:23](https://www.youtube.com/watch?v=5j9JIUuA_cM&t=623)
- Vince Salvino - [15:14](https://www.youtube.com/watch?v=5j9JIUuA_cM&t=914)
- Russell Keith-Magee - [20:24](https://www.youtube.com/watch?v=5j9JIUuA_cM&t=1224)
- Brad Martsberger - [25:19](https://www.youtube.com/watch?v=5j9JIUuA_cM&t=1519)
- Jeremy Gatens - [30:21](https://www.youtube.com/watch?v=5j9JIUuA_cM&t=1821)
- Dane Hillard - [35:31](https://www.youtube.com/watch?v=5j9JIUuA_cM&t=2131)
- Caleb Collins-Parks - [40:27](https://www.youtube.com/watch?v=5j9JIUuA_cM&t=2427)
```

hoping this multiline suggest works

File: _schedule/talks/2019-09-25-12-50-t0-lightning-talks.md [hunk 10]
```diff
@@ -12,8 +12,15 @@ room: Salon A-E
 sitemap: true
 title: Lightning Talks
 track: t0
+video_url: https://www.youtube.com/watch?v=EwEVAs900aE
 ---
 
-Salon A-E
-
-Wednesday sign ups go live Tuesday at 2:20pm and close Wednesday at 9am (please eat breakfast!) with speaker notifications by 11am Wednesday. [Register here!](http://bit.ly/dcus2019lt)
+- Adam Fast - 00:00
```
Comment (drewbrew) [inline _schedule/talks/2019-09-25-12-50-t0-lightning-talks.md]:
```suggestion
- Adam Fast - [00:00](https://www.youtube.com/watch?v=EwEVAs900aE&t=0)
```

File: _schedule/talks/2019-09-25-12-50-t0-lightning-talks.md [hunk 11]
```diff
@@ -12,8 +12,15 @@ room: Salon A-E
...
+- Marshall Sutton - 4:24
```
Comment (drewbrew) [inline _schedule/talks/2019-09-25-12-50-t0-lightning-talks.md]:
```suggestion
- Marshall Sutton - [4:24](https://www.youtube.com/watch?v=EwEVAs900aE&t=264)
```

File: _schedule/talks/2019-09-25-12-50-t0-lightning-talks.md [hunk 12]
```diff
@@ -12,8 +12,15 @@ room: Salon A-E
...
+- Daniele Procida - 9:29
```
Comment (drewbrew) [inline _schedule/talks/2019-09-25-12-50-t0-lightning-talks.md]:
```suggestion
- Daniele Procida - [9:29](https://www.youtube.com/watch?v=EwEVAs900aE&t=569)
```

File: _schedule/talks/2019-09-25-12-50-t0-lightning-talks.md [hunk 13]
```diff
@@ -12,8 +12,15 @@ room: Salon A-E
...
+- Rodrigo Suarez - 14:17
```
Comment (drewbrew) [inline _schedule/talks/2019-09-25-12-50-t0-lightning-talks.md]:
```suggestion
- Rodrigo Suarez - [14:17](https://www.youtube.com/watch?v=EwEVAs900aE&t=857)
```

File: _schedule/talks/2019-09-25-12-50-t0-lightning-talks.md [hunk 14]
```diff
@@ -12,8 +12,15 @@ room: Salon A-E
...
+- Nicolle Cysneiros - 19:07
```
Comment (drewbrew) [inline _schedule/talks/2019-09-25-12-50-t0-lightning-talks.md]:
```suggestion
- Nicolle Cysneiros - [19:07](https://www.youtube.com/watch?v=EwEVAs900aE&t=1147)
```

File: _schedule/talks/2019-09-25-12-50-t0-lightning-talks.md [hunk 15]
```diff
@@ -12,8 +12,15 @@ room: Salon A-E
...
+- Griffith Rees - 23:57
```
Comment (drewbrew) [inline _schedule/talks/2019-09-25-12-50-t0-lightning-talks.md]:
```suggestion
- Griffith Rees - [23:57](https://www.youtube.com/watch?v=EwEVAs900aE&t=1437)
```

File: _schedule/talks/2019-09-25-12-50-t0-lightning-talks.md [hunk 16]
```diff
@@ -12,8 +12,15 @@ room: Salon A-E
...
+- David Vaz - 28:39
```
Comment (drewbrew) [inline _schedule/talks/2019-09-25-12-50-t0-lightning-talks.md]:
```suggestion
- David Vaz - [28:39](https://www.youtube.com/watch?v=EwEVAs900aE&t=1719)
```

File: _schedule/talks/2019-09-25-12-50-t0-lightning-talks.md [hunk 17]
```diff
@@ -12,8 +12,15 @@ room: Salon A-E
...
+- Elizabeth Uselton - 32:52
```
Comment (drewbrew) [inline _schedule/talks/2019-09-25-12-50-t0-lightning-talks.md]:
```suggestion
- Elizabeth Uselton - [32:52](https://www.youtube.com/watch?v=EwEVAs900aE&t=1972)
```

File: _schedule/talks/2019-09-25-12-50-t0-lightning-talks.md [hunk 18]
```diff
@@ -12,8 +12,15 @@ room: Salon A-E
...
+- Ed Rivas - 36:46
```
Comment (drewbrew) [inline _schedule/talks/2019-09-25-12-50-t0-lightning-talks.md]:
```suggestion
- Ed Rivas - [36:46](https://www.youtube.com/watch?v=EwEVAs900aE&t=2206)
```

File: _schedule/talks/2019-09-23-11-30-t0-awesome-automated-apis-with-automagic.md (hunk 1 above)
Comment (jefftriplett) [reply _schedule/talks/2019-09-23-11-30-t0-awesome-automated-apis-with-automagic.md]: Basically https://editorconfig.org/

We have a config that controls how files get saved. Hopefully, someone adds github support and then we don't have to worry about editors doing weird things.

File: _schedule/talks/2019-09-24-12-50-t0-lightning-talks.md (hunk 9 above)
Comment (drewbrew) [inline _schedule/talks/2019-09-24-12-50-t0-lightning-talks.md]:
```suggestion
- Adem Gaygusuz - [00:00](https://www.youtube.com/watch?v=5j9JIUuA_cM&t=0)
```

File: _schedule/talks/2019-09-24-12-50-t0-lightning-talks.md [hunk 19]
```diff
@@ -12,8 +12,15 @@ room: Salon A-E
...
+- Laura Lorenz - 5:24
```
Comment (drewbrew) [inline _schedule/talks/2019-09-24-12-50-t0-lightning-talks.md]:
```suggestion
- Laura Lorenz - [5:24](https://www.youtube.com/watch?v=5j9JIUuA_cM&t=324)
```

File: _schedule/talks/2019-09-24-12-50-t0-lightning-talks.md [hunk 20]
```diff
@@ -12,8 +12,15 @@ room: Salon A-E
...
+- Micah Lyle - 10:23
```
Comment (drewbrew) [inline _schedule/talks/2019-09-24-12-50-t0-lightning-talks.md]:
```suggestion
- Micah Lyle - [10:23](https://www.youtube.com/watch?v=5j9JIUuA_cM&t=623)
```

File: _schedule/talks/2019-09-24-12-50-t0-lightning-talks.md [hunk 21]
```diff
@@ -12,8 +12,15 @@ room: Salon A-E
...
+- Vince Salvino - 15:14
```
Comment (drewbrew) [inline _schedule/talks/2019-09-24-12-50-t0-lightning-talks.md]:
```suggestion
- Vince Salvino - [15:14](https://www.youtube.com/watch?v=5j9JIUuA_cM&t=914)
```

File: _schedule/talks/2019-09-24-12-50-t0-lightning-talks.md [hunk 22]
```diff
@@ -12,8 +12,15 @@ room: Salon A-E
...
+- Russell Keith-Magee - 20:24
```
Comment (drewbrew) [inline _schedule/talks/2019-09-24-12-50-t0-lightning-talks.md]:
```suggestion
- Russell Keith-Magee - [20:24](https://www.youtube.com/watch?v=5j9JIUuA_cM&t=1224)
```

File: _schedule/talks/2019-09-24-12-50-t0-lightning-talks.md [hunk 23]
```diff
@@ -12,8 +12,15 @@ room: Salon A-E
...
+- Brad Martsberger - 25:19
```
Comment (drewbrew) [inline _schedule/talks/2019-09-24-12-50-t0-lightning-talks.md]:
```suggestion
- Brad Martsberger - [25:19](https://www.youtube.com/watch?v=5j9JIUuA_cM&t=1519)
```

File: _schedule/talks/2019-09-24-12-50-t0-lightning-talks.md [hunk 24]
```diff
@@ -12,8 +12,15 @@ room: Salon A-E
...
+- Jeremy Gatens - 30:21
```
Comment (drewbrew) [inline _schedule/talks/2019-09-24-12-50-t0-lightning-talks.md]:
```suggestion
- Jeremy Gatens - [30:21](https://www.youtube.com/watch?v=5j9JIUuA_cM&t=1821)
```

File: _schedule/talks/2019-09-24-12-50-t0-lightning-talks.md [hunk 25]
```diff
@@ -12,8 +12,15 @@ room: Salon A-E
...
+- Dane Hillard - 35:31
```
Comment (drewbrew) [inline _schedule/talks/2019-09-24-12-50-t0-lightning-talks.md]:
```suggestion
- Dane Hillard - [35:31](https://www.youtube.com/watch?v=5j9JIUuA_cM&t=2131)
```

File: _schedule/talks/2019-09-24-12-50-t0-lightning-talks.md [hunk 26]
```diff
@@ -12,8 +12,15 @@ room: Salon A-E
...
+- Caleb Collins-Parks - 40:27
```
Comment (drewbrew) [inline _schedule/talks/2019-09-24-12-50-t0-lightning-talks.md]:
```suggestion
- Caleb Collins-Parks - [40:27](https://www.youtube.com/watch?v=5j9JIUuA_cM&t=2427)
```

**Pull request merged**